### DISCUSSION TITLE
New Category - Fashion

### DISCUSSION THREAD

**Discussion by BLR-0118** [ONDC-Official/protocol-network-extension#54] [Announcements]
Following documents are available for adding the fashion category to ONDC:
>[Fashion Taxonomy](https://docs.google.com/spreadsheets/d/1G4tLF1E0cO4SlLHs95yGlEffwkc3WQZ1/edit#gid=1630620725)

>[Fashion Taxonomy mapping to Protocol](https://docs.google.com/spreadsheets/d/1HSxRqvjMbqAwO4YmUgfXjPnTw_3eK48HyjvoX7XeuAM/edit#gid=1424884528)

>[API contract extension for Fashion](https://docs.google.com/document/d/1Dm5pFvSHKgRxZwgxgOFJZIqAmd31En8p-PIoEku57Es/edit)

**Comment by BLR-0118** [comment]
@get2abhi

**Reply by get2abhi** [reply]
Checking..

**Reply by get2abhi** [reply]
Few important points relevant to the customer

1.  The search should also have sorting options like price, rating and new arrivals
2. Rating attribute(Float) needed in product schema
3. Image quality of Fashion images should have some quality checks like white background, etc. and should be hosted on a CDN for minimal load time.
4. Rating and Reviews API needed
5. Wishlist API Needed
